### DISCUSSION TITLE
Found a project about speech recognition

### DISCUSSION THREAD

**Discussion by despairTK** [Purfview/whisper-standalone-win#292] [General]
Recently I found another project about speech recognition：https://github.com/FunAudioLLM/SenseVoice
In its documentation, it is said that it uses more than 400,000 hours of data training, supports more than 50 languages, and has better recognition performance than the Whisper model. I wonder if you will get involved with this project?

**Comment by Purfview** [comment]
> and has better recognition performance than the Whisper model.

I see questionably better results for Chinese, and it has no timestamps.

>I wonder if you will get involved with this project?

No.
